actix-files/src/files.rs

```diff
@@ -46,24 +47,6 @@ impl fmt::Debug for Files {
     }
 }
 
-impl Clone for Files {
```

Comment:
Is removing the `Clone` impl necessary?

Reply:
the issue is that for support the clone the guards need to be wrapped by an `Rc`, or need to be able to implement the clone, 
the previous implementation had the `Rc` but the `config.register_service(rdef, guards, self, None)` do not support `Rc` guards.

A  way around this would be introduce a new `impl <T:Guard> Guard for Rc<T>`  so then the guards vec can become a 
` Vec<Box<Rc<Guard>>>`  that become clonable and implement itself `Vec<Box<Guard>>`

If this make sense i can add that implementation myself and than re-introduce the clone.
The question is also if the `Clone` is needed other structure like `Resource` do not implement it.

Reply:
Ok Could manage to do that, now the changes looks even smaller then before.